src/adapter.ts

```diff
@@ -262,4 +275,27 @@ export default class JestTestAdapter implements TestAdapter {
   private retireAllTests() {
     this.retireEmitter.fire({});
   }
+
+  private logSuite(suiteOrTest?: TestSuiteInfo|TestInfo, depth: number = 0): void {
+    if (suiteOrTest == null) { return; }
+
+    const indent = (inDepth: number): string => {
+      let spaces = "";
+      for (let i = 0; i < inDepth; i++) {
+        spaces += "  ";
+      }
+      return spaces;
```

Comment:
A similar operation that might be easier to read later:

```suggestion
      return Array(inDepth).fill(' ').join('');
```

Reply:
Nice, I haven't thought of that. Updated.